test/evergreen.yml

```diff
@@ -449,7 +450,7 @@ functions:
 
             # Synchronize the generated documentation with the current one.
             echo "Synchronizing documentation for branch $branch ..."
-            rsync -avq ../wiredtiger/docs-$branch/ $branch/ --delete
+            rsync -avq ../docs-$branch/ $branch/ --delete
```

Comment:
Why are we setting the verbose `-v` and quiet `-q` flags at the same time?

Reply:
Good point, the "-v" flag can be saved.

Reply:
See [e504420](https://github.com/wiredtiger/wiredtiger/pull/9854/commits/e504420c0c0808a7c41d10fde2ccac39fc3b97e6)